app/views/casework/crime_applications/show.html.erb

```diff
@@ -9,7 +9,7 @@
                locals: {
                  applicant: @crime_application.client_details.applicant,
                  nino: @crime_application.formatted_applicant_nino,
-                 applicant_tel: @crime_application.formatted_applicant_telephone_number
+                 applicant_tel: @crime_application.phone_number.for_applicant
```

Comment:
Feels like the order should read, `crime_application.applicant.phone_number` or `crime_application.applicant.formatted_phone_number` or something. This reads like the crime application has a phone number. 

In that case, would it be better to have an applicant presenter? I see there is also a `formatted_applicant_nino` method on line 11 so looks like there are a few methods that could move into that